tests/infer/test_valid_models.py

```diff
@@ -487,6 +487,8 @@ def guide():
 
 
 def test_enum_discrete_global_local_error():
+    if dist.USE_TORCH_DISTRIBUTIONS:
+        pytest.xfail(reason="torch Bernoulli is too permissive?")
```

Comment:
Is this failing because of some broadcasting issue?

Reply:
I believe so. This is a failing death test, so the failure is due to **lack** of an error being raised. Pyro's old distributions did not allow broadcasting and PyTorch does. I think it's safe to put this off until just before release.

Reply:
Sounds good! Just want to know all broadcasting related failures / unexpected outcomes, so that if there is a need to revisit it on the PyTorch side, we can do that since its still early days.